datafusion/proto/build.rs

```diff
@@ -32,7 +32,9 @@ fn main() -> Result<(), String> {
 fn build() -> Result<(), String> {
     use std::io::Write;
 
-    let out = std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap());
+    let out = std::path::PathBuf::from(
+        std::env::var("OUT_DIR").expect("Cannot find OUT_DIR environment vairable"),
```

Comment:
`expect` seems semi-reasonable in tests or `build.rs`

Reply:
Imho expect here gives more user friendly message than `.unwrap` with very generic text

Reply:
expect is a definite improvement to `unwrap` in my opinion too